eng/mgmt/Directory.Build.Mgmt.props

```diff
@@ -1,11 +1,13 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<PropertyGroup>
-		<PkgVersion>0.10.0</PkgVersion>
+		<PkgVersion>0.12.0-dev.20190715.1</PkgVersion>
 		<PkgLocalDir>$(RepoRoot)\restoredPackages\microsoft.internal.netsdkbuild.mgmt.tools\$(PkgVersion)\Sdk</PkgLocalDir>
 	</PropertyGroup>
 
+	<!-- during execution of tasks tests, this conditional import will satisfy the import resolution. And you will have to manually execute a target to restore the package
```

Comment:
Now that I've got the release pipeline working and publishing for these tools can you please go ahead and update this version to the latest published (0.12.0-dev.20190716.14) and delete the package?  If so I'll go ahead and close my PR https://github.com/Azure/azure-sdk-for-net/pull/6879 instead of us fighting which one merges first :)

Reply:
@weshaggard I am going to ask @erich-wang to open a new PR to get rid of the checked in package.
This would be a good way for him to update these tools package on his own.